api/src/main/java/org/openmrs/module/appointmentscheduling/AppointmentResource.java

```diff
@@ -0,0 +1,119 @@
+package org.openmrs.module.appointmentscheduling;
+
+import org.openmrs.BaseOpenmrsData;
+import org.openmrs.Location;
+import org.openmrs.Provider;
+
+import java.sql.Time;
+import java.util.Date;
+import java.util.Set;
+
+public class AppointmentResource extends BaseOpenmrsData {
```

Comment:
Can we add some comments as to what an AppointmentResource is?  (I'm not in love with the name "AppointmentResource" but I haven't come up with a better on yet.  Let me brainstorm some... :)  

The idea is that this defines the "schedule" that a provider is available for dynamic block creation, correct?

Reply:
An appointmentResource is what stores metadata for creating appointment blocks dynamically,  Is AppointmentBlockMetadata a better naming for it ?

Reply:
Just brainstorming, but what about ProviderSchedule or something similar?  Because that's what it generally represents, correct?  A set of days and times when a particular provider is available?

Reply:
That's right, that can work too

Reply:
Let's go with with ProviderSchedule then!